pkg/restapi/issuer/operation/operations.go

```diff
@@ -540,11 +562,28 @@ func (c *Operation) buildInitiateOIDC4CIFlowPage( //nolint:funlen,gocyclo
 		TxID:        parsedResp.TxID,
 		SuccessText: successText.String(),
 		Pin:         pin,
+		Profiles:    c.buildProfileList(profileID),
 	}); err != nil {
 		logger.Errorf(fmt.Sprintf("execute html template: %s", err.Error()))
 	}
 }
 
+func (c *Operation) buildProfileList(currentProfile string) []*profileView {
+	var profileList []*profileView
```

Comment:
Similar comment as before regarding array of pointers - doesn't need to be pointers